*: ensure that auto-generated files have the proper comments at the beginning

knz opened the pull request:
We have two "consumptions points" for them with different requirements:

- https://github.com/golang/go/issues/13560#issuecomment-277804473
- https://github.com/Reviewable/Reviewable/wiki/FAQ#how-do-i-tell-reviewable-that-a-file-is-generated-and-should-not-be-reviewed

Suggested by @tamird in https://github.com/cockroachdb/cockroach/pull/16295#discussion_r131540674

cc @cockroachdb/build

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/17471)
<!-- Reviewable:end -->

tamird commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 11 of 11 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks pending.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/17471#-:-KqrOt6SnS28_W8lV7HZ:bnfp4nl)*
<!-- Sent from Reviewable.io -->

tamird commented:
Reviewed 4 of 4 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/17471)*
<!-- Sent from Reviewable.io -->

knz committed:
*: ensure that auto-generated files have the proper comments at the beginning

We have two "consumptions points" for them with different requirements:

- https://github.com/golang/go/issues/13560#issuecomment-277804473
- https://github.com/Reviewable/Reviewable/wiki/FAQ#how-do-i-tell-reviewable-that-a-file-is-generated-and-should-not-be-reviewed
